app/src/main/java/com/getdefault/task9/DisplayViewModel.kt

```diff
@@ -0,0 +1,25 @@
+package com.getdefault.task9
+
+import androidx.lifecycle.ViewModel
+
+class DisplayViewModel : ViewModel() {
+
+    private val _personList = ArrayList<Person>()
+    val personList: ArrayList<Person>
```

Comment:
@saurav-kalsoor-default why two variables? You could have achieved the same result by doing a private set()